### PR TITLE
Update multi-cpu-architectures.md

### PR DESCRIPTION
**Pull request by michal-at-travisci**
Extending documentation after https://travis-ci.community/t/debugging-docker-run-issue-on-arm64/5491/5

### CODE REVIEW

File: user/multi-cpu-architectures.md [hunk 1]
```diff
@@ -109,3 +109,17 @@ script: docker run my/test #assuming docker image my/test is arm64v8 ready
 {: data-file=".travis.yml"}
 
 You can also have a look at [Using Docker in Builds](user/docker/).
+
+## Security and LXD container
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
## Security and LXD Container
```

File: user/multi-cpu-architectures.md [hunk 2]
```diff
@@ -109,3 +109,17 @@ script: docker run my/test #assuming docker image my/test is arm64v8 ready
...
+
+Please note, that builds run in LXD containers will be denied access to privileged filesystems and paths. This is due to security reasons - a privileged container with write access to e.g. /sys/kernel/debugfs could create a messy situation on LXD host.
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
> Due to security reasons, builds run in LXD containers will be denied access to privileged filesystems and paths - a privileged container with write access to e.g. /sys/kernel/debugfs might muddle an LXD host.
```

File: user/multi-cpu-architectures.md [hunk 3]
```diff
@@ -109,3 +109,17 @@ script: docker run my/test #assuming docker image my/test is arm64v8 ready
...
+
+As a result, for instance a command in `.travis.yaml` like:
+```yaml
+sudo docker run --privileged --rm -t -v /sys/kernel/debug:/sys/kernel/debug:rw
+```
+{: data-file=".travis.yml"}
+
+would result in error.
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
would result in an error.
```

File: user/multi-cpu-architectures.md [hunk 4]
```diff
@@ -109,3 +109,17 @@ script: docker run my/test #assuming docker image my/test is arm64v8 ready
...
+
+See e.g. [Github issue relevant to the topic](https://github.com/lxc/lxd/issues/2661) and [LXD apparmor setup](https://github.com/lxc/lxd/blob/master/lxd/apparmor/apparmor.go) for more details.
```
Comment (Pezi777) [inline user/multi-cpu-architectures.md]:
```suggestion
Also have a look at the [Github issue relevant to the topic](https://github.com/lxc/lxd/issues/2661) and the [LXD apparmor setup](https://github.com/lxc/lxd/blob/master/lxd/apparmor/apparmor.go) for more details.
```

**Comment by Pezi777** [comment]
@michal-at-travisci thank you :-)

**Comment by michal-at-travisci** [comment]
@Pezi777 no no, it's I who thank you for picking this up so quickly ;-)

**Pull request merged**